org/allPRs.ts

```diff
@@ -57,9 +57,9 @@ export const internalLink = async () => {
     }
 }
 
-export const xcodeprojConfiguration = async () => {
-    if (danger.github.thisPR.repo == "macos-browser") {
-        const projectFile = "DuckDuckGo.xcodeproj/project.pbxproj";
+export const xcodeprojConfiguration_macOS = async () => {
```

Comment:
Renaming this to `macOS` so it's clear what this is operating on, we can add an iOS equivalent later if we decide to (though some project file cleanup will be needed first).